pkgs/development/compilers/acme/default.nix

```diff
@@ -2,12 +2,12 @@
 
 stdenv.mkDerivation rec {
   pname = "acme";
-  version = "120";
+  version = "unstable-2020-08-26";
```

Comment:
```suggestion
  version = "295";
```

Reply:
That's the *SVN revision number* of this commit, not a *release version*.

> If a package is **not a release but a commit** from a repository, then the version part of the name must be the date of that (fetched) commit.

The software had actual numbered releases so that'd be double-incorrect.

Reply:
Oh yeah, svn is a thing, too.